src/Dommel/Resolvers.cs

```diff
@@ -78,15 +78,6 @@ public static IEnumerable<ColumnPropertyInfo> Properties(Type type)
 
             return properties;
         }
-        /// <summary>
-        /// Gets the name of the table in the database for the specified type,
-        /// using the configured <see cref="ITableNameResolver"/>.
-        /// </summary>
-        /// <param name="type">The <see cref="Type"/> to get the table name for.</param>
-        /// <param name="connection">The database connection instance.</param>
-        /// <returns>The table name in the database for <paramref name="type"/>.</returns>
-        public static string Table(Type type, IDbConnection connection) =>
-            Table(type, DommelMapper.GetSqlBuilder(connection));
```

Comment:
This is a breaking change! I don't think we should remove this method. There are probably some applications that use it.

Reply:
@lucaxchaves, you're right. My mistake.
I've put the method back in place.